service/user-read/composer.json

```diff
@@ -21,13 +21,13 @@
   ],
   "require": {
     "php": "^7.1",
-    "prooph/common": "4.0.x-dev",
-    "prooph/event-store": "7.0.x-dev",
-    "prooph/pdo-event-store": "1.0.x-dev"
+    "prooph/common": "^4.0",
+    "prooph/event-store": "^7.0",
+    "prooph/pdo-event-store": "^1.0"
```

Comment:
Did you check it? If I revert it it's just stop working, I think there is not such tags in your repos anymore.